src/registry/PartialRegistry.ts

```diff
@@ -87,6 +88,10 @@ export class PartialRegistry extends SynchronousRegistry implements IRegistry {
     }) as WarpCoreConfig[];
   }
 
+  protected getWarpDeploysForIds(_ids: WarpRouteId[]): WarpRouteDeployConfig[] {
+    throw new Error('Method not implemented.');
```

Comment:
I'd expect this to filter and return some member var state like what `getWarpRoutesForIds` does

Reply:
This is kind of hard to implement because the warpRouteId includes the symbol name (`USDC/base-ethereum`), which WarpDeployConfig does not have. We will need to derive from the `config.token`.

Is it worth implementing now as it seems like we may merge both configs soon

Reply:
Ah, k nvm

Reply:
Please leave a comment about this though